### PR TITLE
Removed star-import (except examples and openravepy/__main__.py)

### PR DESCRIPTION
**Pull request by cielavenir**
when `from numpy import *` happens, since 2.0, max, min, round are imported. and numpy.max behavior is different.

```
In [1]: import numpy

In [2]: max(1.0, numpy.float64(2))
Out[2]: np.float64(2.0)

In [3]: numpy.max(1.0, numpy.float64(2))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-5b03d0a46dac> in <module>()
----> 1 numpy.max(1.0, numpy.float64(2))

/home/mujin/mujin/jhbuildappcontroller/install/lib/python3.9/site-packages/numpy/_core/fromnumeric.py in max(a, axis, out, keepdims, initial, where)
   2897     5
   2898     """
-> 2899     return _wrapreduction(a, np.maximum, 'max', axis, None, out,
   2900                           keepdims=keepdims, initial=initial, where=where)
   2901 

/home/mujin/mujin/jhbuildappcontroller/install/lib/python3.9/site-packages/numpy/_core/fromnumeric.py in _wrapreduction(obj, ufunc, method, axis, dtype, out, **kwargs)
     84                 return reduction(axis=axis, out=out, **passkwargs)
     85 
---> 86     return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
     87 
     88 

TypeError: 'numpy.float64' object cannot be interpreted as an integer
```

on numpy 1.25 and 1.26, there is a workaround like https://github.com/numpy/numpy/blob/v1.26.4/numpy/__init__.py#L245 .

But star-import has to be removed for numpy 2.0.

### CODE REVIEW

**Comment by cielavenir** [comment]
https://tiny.mujin.co.jp/5ut6n

**Comment by cielavenir** [comment]
webstack had flakiness, reran https://tiny.mujin.co.jp/qp1s0

**Comment by cielavenir** [comment]
pipeline worked

**Comment by rdiankov** [comment]
@cielavenir Perhaps better if the files were rewritten not to use `from numpy import *`..

File: python/databases/convexdecomposition.py [hunk 1]
```diff
@@ -525,7 +525,7 @@ def testPointsInside(self,points):
                     elif geom.GetType() == KinBody.Link.GeomType.Sphere:
                         insideinds = numpy.less_equal(sum(localpoints**2,1), geom.GetSphereRadius()**2)
                     elif geom.GetType() == KinBody.Link.GeomType.Cylinder:
-                        insideinds = numpy.less_equal(abs(localpoints[:,1]), 0.5*geom.GetCylinderHeight()) and numpy.less_equal(localpoint[:,0]**2+localpoint[:2]**2, geom.GetCylinderRadius()**2)
+                        insideinds = numpy.less_equal(abs(localpoints[:,1]), 0.5*geom.GetCylinderHeight()) and numpy.less_equal(localpoints[:,0]**2+localpoints[:2]**2, geom.GetCylinderRadius()**2)
```
Comment (cielavenir) [inline python/databases/convexdecomposition.py:528]: @rdiankov could you check if this is correct typo fix?

**Comment by cielavenir** [comment]
https://tiny.mujin.co.jp/7rai4

**Comment by cielavenir** [comment]
Some of the databases code are found to be broken:

```
$ python -m flake8 --select=F403,F821,E999,E112,E111 python/databases/
python/databases/visibilitymodel.py:395:30: F821 undefined name 'visibilitytransforms'
python/databases/grasping.py:550:16: F821 undefined name 'boxdelta'
python/databases/grasping.py:551:66: F821 undefined name 'boxdelta'
python/databases/grasping.py:551:92: F821 undefined name 'normalanglerange'
python/databases/grasping.py:552:18: F821 undefined name 'spheredelta'
python/databases/grasping.py:553:69: F821 undefined name 'spheredelta'
python/databases/grasping.py:553:98: F821 undefined name 'normalanglerange'
```

Seems they are not used, so out of this MR scope

@rdiankov please fix them when you have time

**Comment by cielavenir** [comment]
found that I need to fix all test files....

**Comment by cielavenir** [comment]
https://tiny.mujin.co.jp/1yxbf passed, _maybe_ this pull request is ready

**Comment by cielavenir** [comment]
@felixvd @hemangandhi Product team, could you check? I fear if there are some potential issues

**Comment by rdiankov** [comment]
@cielavenir is this tested and ready?

File: python/databases/inversekinematics.py [hunk 2]
```diff
@@ -143,10 +143,8 @@
 __copyright__ = 'Copyright (C) 2009-2012 Rosen Diankov <rosen.diankov@gmail.com>'
 __license__ = 'Apache License, Version 2.0'
 
-if not __openravepy_build_doc__:
-    from numpy import *
-else:
-    from numpy import array
+from numpy import array, float64, double
+from numpy import pi, sqrt, eye, cross, dot, linalg, reshape, mean, median, mod, random, sum, abs, all
```
Comment (hemangandhi) [inline python/databases/inversekinematics.py]: Why are these separate?

Comment (cielavenir) [reply python/databases/inversekinematics.py]: The background is I feared long lines.

There could be three areas: modules, types, and functions/variables.

I still clearly separate lines for openrave modules import (in test/).

However, separating types (eg array/float64) and functions lines does not seem practical.

For this, maybe I can make long lines.

**Comment by cielavenir** [comment]
actually I found that pyANN.py's `import *` cannot be removed as it also works as a proxy - there are some `pyANN.KDTree` in openrave source code and they were regressed (already fixed).

> I fear if there are some potential issues

I meant such situations. Sorry if that was not clear.

But, perhaps, that should be the only issue.

edit: linkstatistics.py does not use pyANN so that import should be removed instead.

**Comment by cielavenir** [comment]
autotester https://tiny.mujin.co.jp/4jr4o

**Comment by cielavenir** [comment]
@rdiankov it is ready now

**Comment by rdiankov** [comment]
thanks~

**Pull request merged**